### DISCUSSION TITLE
Change the source code like title does not reflect in the UI

### DISCUSSION THREAD

**Discussion by code-with-bhavin** [Budibase/budibase#13655] [General]
When i working on localhost:10000 port and changes in source code does not reflect can you please help to figure out?

i am using windows PC
Chrome Browser

**Comment by mikesealey** [comment]
Hi @bsbhavinsarvaiya 

What is it, exactly, that you're trying to do? Please provide as much information as possible.

**Reply by code-with-bhavin** [reply]
When i working on localhost:10000 port and changes like Login Button Name it's not reflecting in the UI.
i have also tried docker-compose up and docker-compose down command.
